Backend-Roadmap/Controllers/ProjectsController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using BackEnd.Context;
+using BackEnd.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace BackEnd.Controllers
+{
+    namespace BackEnd.Controllers
+    {
+        [Route("api/[controller]")]
+        [ApiController]
+        public class ProjectController : ControllerBase
+        {
+            private readonly AppDbContext _context;
+
+            public ProjectController(AppDbContext context)
+            {
+                _context = context;
+            }
+
+            // GET: api/Project
+            [HttpGet]
+            public async Task<ActionResult<IEnumerable<Project>>> GetProjects()
+            {
+                return await _context.Project.ToListAsync();
+            }
+
+            // GET: api/Project/5
+            [HttpGet("{id}")]
+            public async Task<ActionResult<Project>> GetProject(int id)
+            {
+                var project = await _context.Project.FindAsync(id);
+
+                if (project == null)
+                {
+                    return NotFound();
+                }
+
+                return project;
+            }
+
+            // POST: api/Project
+            [HttpPost]
+            public async Task<ActionResult<Project>> PostProject(Project project)
```

Comment:
Give a name to the route
[HttpPost("CreateProject")]
PostProject([FromBody] Project project)